manim/mobject/svg/brace.py

```diff
@@ -320,7 +336,7 @@ def __init__(
         point_1: Point3DLike | None,
         point_2: Point3DLike | None,
         direction: Vector3D | None = ORIGIN,
```

Comment:
If any of `point_1` and `point_2` is `None`, the code below raises an error. `direction` should also be different from `None`.
```suggestion
        point_1: Point3DLike,
        point_2: Point3DLike,
        direction: Point3DLike = ORIGIN,
```